specificatie/zoek-personen.yaml

```diff
@@ -0,0 +1,333 @@
+  schemas:
+    InclusiefOverledenPersonen:
+      description: |
+        Als je ook overleden personen in het antwoord wilt, geef dan de parameter inclusiefOverledenPersonen op met waarde True.  Zie [functionele specificaties](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.3.0/features/overleden_personen.feature)
+      type: boolean
+      example: true
+    PersonenQuery:
+      type: object
+      required:
+        - type
+        - fields
+      discriminator:
+        propertyName: type
+        mapping:
+          ZoekMetGeslachtsnaamEnGeboortedatum: '#/components/schemas/ZoekMetGeslachtsnaamEnGeboortedatum'
+          ZoekMetNaamEnGemeenteVanInschrijving: '#/components/schemas/ZoekMetNaamEnGemeenteVanInschrijving'
+          RaadpleegMetBurgerservicenummer: '#/components/schemas/RaadpleegMetBurgerservicenummer'
+          ZoekMetPostcodeEnHuisnummer: '#/components/schemas/ZoekMetPostcodeEnHuisnummer'
+          ZoekMetStraatHuisnummerEnGemeenteVanInschrijving: '#/components/schemas/ZoekMetStraatHuisnummerEnGemeenteVanInschrijving'
+          ZoekMetNummeraanduidingIdentificatie: '#/components/schemas/ZoekMetNummeraanduidingIdentificatie'
+      properties:
+        type:
+          type: string
+        fields:
+          $ref: 'filter.yaml#/components/schemas/Fields'
+        gemeenteVanInschrijving:
+          $ref: 'verblijfplaats.yaml#/components/schemas/GemeenteVanInschrijving'
+    RaadpleegMetBurgerservicenummer:
+      required:
+        - burgerservicenummer
+      allOf:
+        - $ref: '#/components/schemas/PersonenQuery'
+        - type: object
+          properties:
+            burgerservicenummer:
+              type: array
+              items:
+                $ref: 'persoon.yaml#/components/schemas/Burgerservicenummer'
+              minItems: 1
+              maxItems: 20
+    ZoekMetGeslachtsnaamEnGeboortedatum:
+      required:
+        - geslachtsnaam
+        - geboortedatum
+      allOf:
+        - $ref: '#/components/schemas/PersonenQuery'
+        - type: object
+          properties:
+            inclusiefOverledenPersonen:
+              $ref: '#/components/schemas/InclusiefOverledenPersonen'
+            geboortedatum:
+              description: |
+                Je kunt alleen zoeken met een volledig geboortedatum. Zie [functionele specificaties](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.3.0/features/parametervalidatie.feature)
+              type: string
+              format: date
+              example: "1964-09-24"
+            geboorteplaats:
+              description: |
+                Gemeentenaam of een buitenlandse plaats of een plaatsbepaling, die aangeeft waar de persoon is geboren. **Zoeken met tekstvelden is [case-Insensitive](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.3.0/features/case_insensitive.feature).**
+              type: string
+              pattern: ^(\?+|\*)?[a-zA-Z0-9À-ÿ '\-\']{1,40}(\?+|\*)?$
```

Comment:
hier zitten wildcards in het pattern. zoeken met geboorteplaats kan niet met wildcard